pages/docs/guides/delayed-functions.mdx

```diff
@@ -2,7 +2,7 @@ import { CodeGroup } from "src/shared/Docs/mdx";
 
 # Enqueueing future jobs
 
-You can easily enqueue jobs in the future with Inngest. Inngest offers two ways to run jobs in the future: delaying jobs for a specific amount of time (eg. run in 1 day), or running code at a specific date and time. There are some benefits to enqueuing jobs using Inngest:
+You can easily enqueue jobs in the future with Inngest. Inngest offers two ways to run jobs in the future: delaying jobs for a specific amount of time (up to a year, and for free plan up to seven days), or running code at a specific date and time. There are some benefits to enqueuing jobs using Inngest:
```

Comment:
We can add this copy on the limits page then link to pricing from the limits page. https://www.inngest.com/docs/usage-limits/inngest

I'm thinking the limits page says "Sleep, delays, etc. up to X for paid plans and Y for free plans" Check the pricing page for more information.

Reply:
Great suggestion, see https://github.com/inngest/website/pull/821/files#r1661356650